sdk/src/types/block/output/basic.rs

```diff
@@ -358,6 +358,12 @@ impl BasicOutput {
     }
 }
 
+impl WorkScore for BasicOutput {
+    fn work_score(&self, work_score_params: WorkScoreParameters) -> u32 {
+        work_score_params.output() + self.features().work_score(work_score_params)
```

Comment:
Unlock conditions not needed? Also for the other outputs

Reply:
yeah ... it looked through the TIPs and couldn't find anything.